### PR TITLE
Update Workspace dashboard,report and account report

### PR DESCRIPTION
**Pull request by vkumbha**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: dashboards/workspace/workspace_account_report.sp [hunk 1]
```diff
@@ -43,78 +31,127 @@ dashboard "workspace_account_report" {
         display = "none"
       }
```
Comment (khushboo9024) [inline dashboards/workspace/workspace_account_report.sp:33]: The `workspace` column is set to `display = "none"`, but we don't have a `workspace` column; instead, we have `Workspace` column. Do we still need it?"

File: dashboards/workspace/workspace_account_report.sp [hunk 2]
```diff
@@ -43,78 +31,127 @@ dashboard "workspace_account_report" {
...
+            metadata {
+              stats {
+                total
+              }
+            }
+          }
+        }
+      }
+    }'
+  order by "Workspace", "Resources" desc
```
Comment (khushboo9024) [inline dashboards/workspace/workspace_account_report.sp:155]:
```suggestion
  order by 
     "Workspace", "Resources" desc;
```

File: dashboards/workspace/workspace_dashboard.sp [hunk 3]
```diff
@@ -157,18 +80,151 @@ query "workspace_count" {
...
+      accounts: resources(filter: "resourceTypeId:tmod:@turbot/aws#/resource/types/aws,tmod:@turbot/azure#/resource/types/azure,tmod:@turbot/gcp#/resource/types/gcp,tmod:@turbot/servicenow#/resource/types/serviceNow,tmod:@turbot/turbot#/resource/types/folder,tmod:@turbot/turbot#/resource/types/file,tmod:@turbot/turbot#/resource/types/smartFolder") {
+        metadata {
+          stats {
+            total
+          }
+        }
+      }
+    }'
+  group by
+    _ctx ->> 'connection_name'
```
Comment (khushboo9024) [inline dashboards/workspace/workspace_dashboard.sp:228]:
```suggestion
    _ctx ->> 'connection_name';
```

File: dashboards/workspace/workspace_dashboard.sp [hunk 4]
```diff
@@ -13,140 +13,63 @@ dashboard "workspace_dashboard" {
     title = "Workspace Statistics"
 
     card {
-      sql   = query.workspace_count.sql
-      width = 3
+      sql   = query.workspaces_count.sql
+      width = 2
```
Comment (khushboo9024) [inline dashboards/workspace/workspace_dashboard.sp:17]: @vkumbha I think the card width should be `3`. Is there any particular reason why we changed it to `2`?

File: dashboards/workspace/workspace_dashboard.sp [hunk 5]
```diff
@@ -157,18 +80,151 @@ query "workspace_count" {
...
+      accounts: resources(filter: "resourceTypeId:tmod:@turbot/turbot#/resource/interfaces/accountable level:self") {
+        metadata {
+          stats {
+            total
+          }
+        }
+      }
+    }'
+  group by
+    _ctx ->> 'connection_name'
```
Comment (khushboo9024) [inline dashboards/workspace/workspace_dashboard.sp:160]:
```suggestion
    _ctx ->> 'connection_name';
```
Please update all the queries wherever applicable.

File: dashboards/workspace/workspace_account_report.sp [hunk 6]
```diff
@@ -43,78 +31,127 @@ dashboard "workspace_account_report" {
...
-      workspace,
-      trunk_title;
+    workspace as "Workspace",
+    (accountables -> 'descendants' -> 'metadata' -> 'stats' -> 'total')::int as "Resources",
+    (accountables -> 'policySettings' -> 'metadata' -> 'stats' ->> 'total')::int as "Policy Settings",
+    (accountables -> 'alerts' -> 'metadata' -> 'stats' ->> 'total')::int as "Alerts",
+    (accountables -> 'activeControls' -> 'metadata' -> 'stats' ->> 'total')::int as "Active Controls"
+  from
+    guardrails_query,
+    jsonb_array_elements(output ->  'resources' -> 'items') as accountables
```
Comment (khushboo9024) [inline dashboards/workspace/workspace_account_report.sp:105]:
```suggestion
    jsonb_array_elements(output -> 'resources' -> 'items') as accountables
```

File: dashboards/workspace/workspace_dashboard.sp (hunk 4 above)
Comment (vkumbha) [reply dashboards/workspace/workspace_dashboard.sp]: You are right. updated the width to 3.

File: dashboards/workspace/workspace_account_report.sp (hunk 1 above)
Comment (vkumbha) [reply dashboards/workspace/workspace_account_report.sp]: not required, removed it. Thanks for catching this!

**Review by khushboo9024 — APPROVED** [review]

**Pull request merged**